apps/explorer/package.json

```diff
@@ -64,6 +64,7 @@
         "react-error-boundary": "^3.1.4",
         "react-hook-form": "^7.43.9",
         "react-hot-toast": "^2.4.0",
+        "react-json-view": "^1.21.3",
```

Comment:
Don't we already have a json viewer that we use? Can we be consistent in this rather than having two ways that json is presented?

Reply:
Do we have another package for json view? I am unable to find it